sds_data_manager/lambda_code/efs_lambda/lambda_function.py

```diff
@@ -0,0 +1,140 @@
+import os
+import shutil
+
+import boto3
+
+# Define the paths
+mount_path = os.environ.get("EFS_MOUNT_PATH")
+
+attitude_symlink_path = f"{mount_path}/latest_attitude_kernel.ah.a"
```

Comment:
What date format do we get these from our partners in? If the date is more like this: `attitude_kernel_YYYYMMDD.ah.a` I would think we might want to change this to be `attitude_kernel_latest.ah.a` to put `latest` in the date area.

Reply:
filenaming convention looks like this for attitude kernel
```
Attitude History Report:

File Name: imap_yyyy_doy_yyyy_doy_##.ah.bc and
    imap_yyyy_doy_yyyy_doy_##.ah.a
    (where yyyy_doy defines the time span of
    the data in the file)

Server Location: imap/data_products/attitude_history/yyyy/

Attitude Predict Report
File Name: imap_pred_yyyy_doy_yyyy_doy_##.ah.bc or
    imap_pred_yyy_doy_yyyy_doy_##.ah.a
    (where yyyy_doy defines the time span
    of the data in the file)

Server Location: imap/data_products/attitude_history/yyyy/

Attitude Metakernel
File Name: imap_yyyy_doy_a##.spice.mk or
    imap_yyyy_doy_a##.stk_a.mk (where yyyy_doy
    is the day of creation)

```

Reply:
Let me know if you still want date in the symlink file. I personally like to avoid date in the symlink name. When we get real file that symlink is pointing, it should have date in it and that file should be what's written to the CDF metadata.

Reply:
Sorry for the confusion, I agree with you here that we definitely don't want a date in the symlink name. I was trying to say where the `latest` would go in the name, first like you have it or just replaying the YYYY date format.
`imap_yyyy_doy_a##.stk_a.mk` -> `imap_latest.stk_a.mk`

I think what you have is good, and it is a descriptive name :+1: